src/runtime/stream.rs

```diff
@@ -105,14 +109,6 @@ impl AsyncTcpStream {
         Ok(stream.into())
     }
 
-    pub(crate) async fn connect_socket_addr(
```

Comment:
This was only used for the trust-dns custom implementation, so it isn't needed anymore